File: 백준/Bronze/2525. 오븐 시계/오븐 시계.py
H, M = map(int, input().split())
time = int(input())

x = (M + time) // 60
y = (M + time) % 60
if x > 0 and (H + x) < 24:
    print(f"{H + x} {y}")
elif x > 0 and (H + x) >= 24:
    print(f"{H + x - 24} {y}")
elif x <= 0:
    print(f"{H} {y}")

File: 백준/Bronze/2525. 오븐 시계/오븐 시계.cc
#include <iostream>
using namespace std;

int main() {
	int H, M;
	int time;
	cin >> H >> M;
	cin >> time;

	int x = (M + time) / 60;
	int y = (M + time) % 60;

	if (x > 0 && (H + x) < 24) {
		cout << H + x << " " << y << endl;
	}
	else if (x > 0 && (H + x) >= 24){
		cout << (H + x - 24) << " " << y << endl;
	}
	else if (x <= 0) {
		cout << H << " " << y << endl;
	}
	return 0;
}
